src/modules/didRegistry/didRegistry.validation.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  DIDAttribute,
+  Encoding,
+  PubKeyType,
+} from '@ew-did-registry/did-resolver-interface';
+import { KeyType } from '@ew-did-registry/keys';
+import { createCheckers, iface, enumtype } from 'ts-interface-checker';
+
+const IS_REQ_STRING = 'string';
+export const { UpdateServicePoint, UpdateDelegate, UpdatePublicKey } =
+  createCheckers({
+    UpdateServicePoint: iface([], {
+      did: IS_REQ_STRING,
+      didAttribute: enumtype({ ServicePoint: DIDAttribute.ServicePoint }),
+      data: iface([], {
+        type: enumtype({ ServicePoint: DIDAttribute.ServicePoint }),
+        value: iface([], {
+          id: IS_REQ_STRING,
+          hash: IS_REQ_STRING,
+          hashAlg: IS_REQ_STRING,
+        }),
+      }),
+    }),
+
+    UpdateDelegate: iface([], {
+      did: IS_REQ_STRING,
```

Comment:
Is it possible to specify DID pattern here?

Reply:
I looked through whole interface and could not find any 'pattern match' or 'regex' or even 'custom validator' .